### DISCUSSION TITLE
which version support docker client api version 1.40？

### DISCUSSION THREAD

**Discussion by trgfree** [chaos-mesh/chaos-mesh#4551] [Q&A]
my k8s's docker client api version is 1.40, when i do an experience of stresschaos, it report an error : "error response from daemon:client version 1.41 is too new.maximum supported api version is 1.40"

**Comment by STRRL** [comment]
hope this doc could help: https://chaos-mesh.org/docs/faqs/#daemon-throws-an-error-similar-to-version-141-is-too-new-the-maximum-supported-api-version-is-139
